### PR TITLE
feat(python3): support for python 3

### PR DESCRIPTION
**Pull request by daddykotex**
I don't know if this is the way to go to support the two versions with the same code base. What do you think?

### CODE REVIEW

**Comment by daddykotex** [comment]
This is showing up in the unittests for `python3` and I can't figure out why:

```
sys:1: ResourceWarning: unclosed <socket.socket fd=4, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 37553)>
```

File: ittests/it_test.py [hunk 1]
```diff
@@ -1,21 +1,31 @@
...
-        key = Config.get("Credentials", "key")
-        secret = Config.get("Credentials", "secret")
+        if PY3:
+            import configparser
+            myconfig = configparser.ConfigParser()
+            myconfig.read("ittests/creds.ini")
+            key = myconfig.get("Credentials", "key")
+            secret = myconfig.get("Credentials", "secret")
+        else:
+            import ConfigParser
```
Comment (mvonthron) [inline ittests/it_test.py]: I'd do a `import ConfigParser as configparser` and leave the common code as is.

File: ittests/it_test.py [hunk 2]
```diff
@@ -92,33 +102,4 @@ def search_object_query(deviceId):
               "eq": deviceId
             }
           }
-        }
-
-    @staticmethod
-    def eventually_assert(myAssert):
```
Comment (mvonthron) [inline ittests/it_test.py]: not that I'm complaining but why did you remove them?

Comment (daddykotex) [inline ittests/it_test.py]: Tests were running in 10 minutes instead of 30 seconds, I got tired.

Also, I began to remove it from all of our clients' tests as the client code already throws an error if the post/put/get/whatever fails. Checking in restitution is an extra check and I don't think it adds any value at all.

File: ittests/it_test.py [hunk 3]
```diff
@@ -1,21 +1,28 @@
...
     @staticmethod
     def getClient():
-        Config = ConfigParser.ConfigParser()
-        Config.read("ittests/creds.ini")
-        key = Config.get("Credentials", "key")
-        secret = Config.get("Credentials", "secret")
+        if PY3:
+          from configparser import ConfigParser
+        else:
+          from ConfigParser import ConfigParser
```
Comment (daddykotex) [inline ittests/it_test.py]: @mvonthron Did that instead. Note that the package was renamed in py3, I have to do it

**Comment by jslecointre** [comment]
regression tests perfomed for ingestion for owners objects events and for restitution in python 2.7.

Restitution was also successfully tested in python 3.6.
